src/themes/admin_future/modules/voting/setting.tpl

```diff
@@ -0,0 +1,24 @@
+<!-- BEGIN: main -->
+<div class="card border-primary border-3 border-bottom-0 border-start-0 border-end-0">
+    <div class="card-body pt-4">
+        <form method="post" class="ajax-submit" action="{$smarty.const.NV_BASE_ADMINURL}index.php?{$smarty.const.NV_LANG_VARIABLE}={$smarty.const.NV_LANG_DATA}&amp;{$smarty.const.NV_NAME_VARIABLE}={$MODULE_NAME}&amp;{$smarty.const.NV_OP_VARIABLE}={$OP}" novalidate>
+            <div class="row mb-3">
+                <label class="col-sm-5 col-lg-3 col-form-label text-sm-end">
+                    {$LANG->getModule('difftimeout')}
+                </label>
+                <div class="col-sm-3 col-lg-3 col-xxl-2">
+                    <div class="input-group">
+                        <input type="text" class="form-control" name="difftimeout" value="{$DATA.difftimeout}">&nbsp;<span class="d-flex align-items-center">{$LANG->getModule('hours')}</span>
+                    </div>
```

Comment:
@nambmtk1993 cấu trúc input-group này sai rồi em. Với lại thời gian nó dạng số thì em để `type="number"`

Reply:
@nambmtk1993 thẻ input này em không có label nào liên kết với nó, cái 

```html
<div class="col-sm-5 col-lg-3 col-form-label text-sm-end">
                    {$LANG->getModule('difftimeout')}
                </div>
```

Nên đổi thành label, thêm for = ID thẻ input. Nếu không thì thẻ input cần có aria-label, ngoài ra cần có aria-describedby cho thẻ input, liên kết với cái input-group-text nhằm giải nghĩa đơn vị cho nó. Xem các HTML mẫu trong https://getbootstrap.com/docs/5.3/forms/input-group/

Reply:
@nambmtk1993 chỗ này về khả năng tiếp cận nó đang không tốt. Phải như này nó mới chuẩn em:

1. Thẻ này `<div class="col-sm-5 col-lg-3 col-form-label text-sm-end">` đổi thành label, thêm `for="difftimeout"`
2. Thẻ input thêm `id="difftimeout" aria-describedby="difftimeout-addon"`
3. Thẻ `<span class="input-group-text">` nên cho nó xuống dòng và thêm `id="difftimeout-addon"`